mava/components/updating/checkpointer.py

```diff
@@ -51,12 +53,21 @@ def on_parameter_server_init(self, server: SystemParameterServer) -> None:
             None.
         """
         saveable_parameters = SaveableWrapper(server.store.parameters)
+        old_trainer_steps = server.store.parameters["trainer_steps"].copy()
         server.store.system_checkpointer = acme_savers.Checkpointer(
             object_to_save=saveable_parameters,  # must be type saveable
             directory=server.store.experiment_path,
             add_uid=False,
             time_delta_minutes=0,
         )
+
+        # Check if the checkpointer restore the network
+        # and if the user want network with best performance.
```

Comment:
Check if the checkpointer restored the network parameters and if the user wants the network with the best performance.